AlphaWallet/Transfer/Coordinators/TransactionConfirmationCoordinator.swift

```diff
@@ -18,8 +18,9 @@ protocol TransactionConfirmationCoordinatorDelegate: CanOpenURL, SendTransaction
 
 class TransactionConfirmationCoordinator: Coordinator {
     private let configuration: TransactionType.Configuration
-    private lazy var viewModel: TransactionConfirmationViewModel = .init(configurator: configurator, configuration: configuration, assetDefinitionStore: assetDefinitionStore, domainResolutionService: domainResolutionService, tokensService: tokensService)
+
     private lazy var rootViewController: TransactionConfirmationViewController = {
+        let viewModel = TransactionConfirmationViewModel(configurator: configurator, configuration: configuration, assetDefinitionStore: assetDefinitionStore, domainResolutionService: domainResolutionService, tokensService: tokensService)
```

Comment:
What was leaking?

Reply:
View model was capturing stores publisher that captures view model

https://github.com/AlphaWallet/alpha-wallet-ios/pull/5718/commits/986980a2d3cda102759dab1806c86554c07f00a0#:~:text=.flatMap%20%7B%20_%20in%20self.token%20%7D